library/src/main/java/de/agilecoders/wicket/webjars/collectors/AssetsMap.java

```diff
@@ -32,19 +32,29 @@
     private final SortedMap<String, String> fullPathIndex;
     private final AssetPathCollector[] collectors;
     private final String recentVersionPlaceHolder;
+    private static AssetsMap assetsMap;
```

Comment:
It's some kind of more dynamic substitution of what static Holder and it's static member do. I fully agree with you that it can be done even better, but @WeaxMe tried to fix with minimal changes.